index.html

```diff
@@ -52,4 +52,114 @@ <h1 class="intro height">I’m Kartikay Glad to see you!</h1>
       </ul>
     </div>
   </section>
+  <section>
+    <div class="grid-container">
+      <div class="img-container">
+        <img class="work-image" src="./Images/tonic-one.png" alt="work-Image" />
+      </div>
+      <div class="card">
+        <div class="card-element">
+          <h2 class="card-title">Tonic</h2>
+          <div class="card-section">
+            <p class="para-one">CANOPY</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">Back End Dev</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">2015</p>
+          </div>
+          <p class="card-body">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tech-stack">
+            <button class="tech-items" type="button">html</button>
+            <button class="tech-items" type="button">css</button>
+            <button class="tech-items" type="button">javascript</button>
+          </div>
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </div>
```

Comment:
- [ ] Good job so far 👍 but in the project cards section, because your project image is in a different `div` tag from the rest of the project information, its width is also different. I suggest putting the project image inside the same `div` tag as the rest of the information concerning that project, in this case, it will be better to put the image inside the `div` of class name `card-element`. Below is a view of your page at 375px . Notice how the image container width is smaller but in the Figma template, it is supposed to be the same.
- Your design:
![Capture55](https://user-images.githubusercontent.com/85113567/153697431-e04d1713-b851-4e98-952c-09b25291f319.PNG)
- Figma design:

![Capture56](https://user-images.githubusercontent.com/85113567/153697504-67c7544b-828f-4354-905a-c67ad270e72c.PNG)

Reply:
Thanks for the valuable suggestion. I have implemented all the changes.